src/components/CloseButton.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { StyleProp, StyleSheet, View, ViewStyle } from 'react-native'
+import Touchable from 'src/components/Touchable'
+import Times from 'src/icons/Times'
+import { navigateBack } from 'src/navigator/NavigationService'
+import { Spacing } from 'src/styles/styles'
+
+interface Props {
+  style?: StyleProp<ViewStyle>
+  size?: number
+  testID?: string
+}
+
+export default function CloseButton({ style, size, testID }: Props) {
+  const onPress = () => {
+    navigateBack()
+  }
+
+  return (
+    <View style={styles.container}>
+      <Touchable
+        testID={testID}
+        onPress={onPress}
+        style={[style, styles.button]}
+        borderRadius={Spacing.Thick24}
```

Comment:
this should probably be a function of size ideally? we probably won't pass a higher size for icon, but something to consider

Reply:
I played around a bit with abstracting the buttons to a shared component and ultimately decided against it in this PR and just focused on the touchable fix. The `CloseButton.tsx`, `AccountCircleButton.tsx`, `QrScanButton.tsx` and `NotificationBell.tsx` buttons should probably be refactored to use a shared component and I agree making the borderRadius a function of size would make sense there.